g/gziphandler/gziphandler_ubi_8.3.sh

```diff
@@ -0,0 +1,62 @@
+#----------------------------------------------------------------------------
+#
+# Package               : gziphandler 
+# Version               : 1.1.1,1.1.0
+# Source repo           : https://github.com/nytimes/gziphandler
+# Tested on             : UBI 8.3
+# Script License        : Apache License, Version 2 or later
+# Passing Arguments     : Passing Arguments: 1.Version of package,
+# Script License        : Apache License, Version 2 or later
+# Maintainer            : Arumugam N S<asellappen@yahoo.com>/Priya Seth<sethp@us.ibm.com>
+#
+# Disclaimer            : This script has been tested in non-root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+if [ -z "$1" ]; then
+  export VERSION=master
+else
+  export VERSION=$1
+fi
+
+if [ -d "gziphandler" ] ; then
+  rm -rf gziphandler
+fi
+
+# Dependency installation
+sudo dnf install -y git golang 
+# Download the repos
+git clone https://github.com/nytimes/gziphandler
+
+
+# Build and Test gziphandler
+cd gziphandler 
+git checkout $VERSION
+ret=$?
+if [ $ret -eq 0 ] ; then
+ echo "$Version found to checkout "
+else
+ echo "$Version not found "
+ exit
+fi
+
+#Build and test
+go get -v -t ./...
+ret=$?
+if [ $ret -ne 0 ] ; then
+  echo "Build failed "
+else
+  go test -v ./...
+  if [ $ret -ne 0 ] ; then
```

Comment:
We are not capturing any ret value here, but checking it's status ? Rest all LGTM

Reply:
good catch ,missed and now taken care